shoop/core/templatetags/shoop_common.py

```diff
@@ -23,6 +24,14 @@
 from shoop.utils.serialization import ExtendedJSONEncoder
 
 
+@library.global_function
+def get_localized_language_name(code, name):
+    if code == name:
+        return get_language_info(code.lower())["name_local"]
```

Comment:
I think it would be good idea to include them both in the language switcher, i.e. when current language is English, the choices would be:
 * Chinese (Simplified) (中文)
 * **English (English)**
 * Finnish (suomi)
 * Japanese (日本語)

@juhakujala As an UX expert, what do you think?